src/Microsoft.ServiceFabric.Actors/Runtime/ActorStateProviderHelper.cs

```diff
@@ -274,7 +274,7 @@ internal static IActorStateProvider CreateDefaultStateProvider(ActorTypeInformat
                 stateProvider = new VolatileActorStateProvider();
             }
 
-            // Get state provider override from settings if specified, used by tests to override state providers.
+            // Gets the state provider override from settings if specified, used by tests to override state providers.
```

Comment:
This is a code comment telling the follow of the code, this change is not needed.